src/components/TextField/TextField.style.ts

```diff
@@ -1,32 +1,82 @@
 import { css, SerializedStyles } from '@emotion/core';
-import { rem, transparentize } from 'polished';
-import { Theme } from '../../theme';
+import { rem } from 'polished';
 import { Props } from './TextField';
+import { Theme } from '../../theme';
 
+const wrapperStyleSwitch = (theme: Theme, lean?: boolean, error?: boolean, styleType?: string) => {
+  if (lean) {
+    return `
+      background-color: transparent;
+      box-shadow: inset 0 0 0 1px transparent;
+    `;
+  }
+
+  switch (styleType) {
+    case 'outlined':
+      return `
+        background-color: white;
+        box-shadow: inset 0 0 0 1px
+          ${error ? theme.palette.error[400] : '#dfdfdf'};
```

Comment:
Colors that do not match our color palette, have been replaced according to designs!

Reply:
Here the palette colors should be used. 

If the designs have different colors than the ones in palette we should discuss it with the design team to either change the palette yet again or update the designs.

Reply:
Sure @Christos-Zacharopoulos !
We will sync with the design team in order to find a solution that fits both sides.
For now I am leaving the components as they are and will add a todo comment (as you suggested) in order to keep track of the future required changes!